lib/Backend/BackendUtils.cpp

```diff
@@ -322,11 +322,10 @@ runtime::RuntimeBundle runtime::RuntimeBundle::create(const Function &F) {
                                 /*activationsMaxSize*/ 0);
 }
 
-runtime::RuntimeBundle
-runtime::RuntimeBundle::create(const IRFunction &F,
-                               MemoryAllocator &constantAllocator,
-                               MemoryAllocator &placeholderAllocator,
-                               MemoryAllocator &activationsAllocator) {
+runtime::RuntimeBundle runtime::RuntimeBundle::create(
+    const IRFunction &F, MemoryAllocator &constantAllocator,
+    MemoryAllocator &placeholderAllocator,
+    MemoryAllocator &activationsAllocator, bool contiguous) {
```

Comment:
Can we infer the value of `contiguous` without depending on the caller to pass the correct value in? i.e.
```
bool contiguous = (&constantAllocator == &placeholderAllocator) && (&constantAllocator == &activationsAllocator);
```
or something like that?

Reply:
We could, do you think its clearer?

Reply:
I don't think it's clearer by itself without any explanation, but I think a one or two line comment above the line I suggested would suffice.

My concern is that this overload of `RuntimeBundle::create` can still be called like so
```
auto r = runtime::RuntimeBundle::create(F, allocator, allocator, allocator);
```
and the memory usage will be 2-3x as much as needed.

Reply:
OK sure, that makes sense. There's a limit to how much we can guard against people using interfaces wrong though.